tck/src/main/java/org/eclipse/microprofile/lra/tck/TckParticipantTests.java

```diff
@@ -169,7 +180,42 @@ public void testNonJaxRsCompletionStageResponseAndParticipantStatus() throws Int
 
         assertTrue("Non JAX-RS @Status method with CompletionStage<ParticipantStatus> should have been called",
             lraMetricService.getMetric(LRAMetricType.Status, lraId) >= 1);
-        
+
         lraTestService.waitForRecovery(lraId);
     }
+
+    @Test
+    public void cancelLraDuringBusinessMethod() throws InterruptedException, ExecutionException {
+        ExecutorService es = Executors.newSingleThreadExecutor();
+        LRAClientOps lraOps = lraTestService.getLRAClient();
+        URI lraId = lraOps.startLRA(null, lraClientId(), 0L, ChronoUnit.MILLIS);
+        LOGGER.info(String.format("Started LRA with URI %s", lraId));
+        // Start business method asynchronously, i.e. return immediately.
+        Future<Response> lraFuture = es.submit(() -> tckSuiteTarget.path(LongBusinessMethodParticipant.ROOT_PATH)
+                .path(LongBusinessMethodParticipant.BUSINESS_METHOD)
+                .request()
+                .header(LRA.LRA_HTTP_CONTEXT_HEADER, lraId)
+                .put(Entity.text("")));
```

Comment:
you could use `async()` to handle async REST invocation. - https://javaee.github.io/javaee-spec/javadocs/javax/ws/rs/client/Invocation.Builder.html#async--

Reply:
What would the benefit be (this is a TCK test)

Reply:
+1

Reply:
That's a good point, I've incorporated `acync()` into the test.